### PR TITLE
Topic changelog v1.6.0

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- [x] depends on #163

### CODE REVIEW

File: lib/isaac/isaac_version.hpp [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #pragma once
 
 #define ISAAC_VERSION_MAJOR 1
-#define ISAAC_VERSION_MINOR 6
-#define ISAAC_VERSION_PATCH 0
+#define ISAAC_VERSION_MINOR 5
+#define ISAAC_VERSION_PATCH 2
```
Comment (PrometheusPi) [inline lib/isaac/isaac_version.hpp]: @psychocoderHPC I think, here went something wrong - shouldn't that change from 1.5.2 to 1.6.0 - but seems to do the inverse

**Review by PrometheusPi — CHANGES_REQUESTED** [review]

File: lib/isaac/isaac_version.hpp (hunk 1 above)
Comment (sbastrakov) [reply lib/isaac/isaac_version.hpp]: Yes, may be a mistake during rebase with wrong version used

**Comment by PrometheusPi** [comment]
@psychocoderHPC Could you comment on this issue?

File: lib/isaac/isaac_version.hpp (hunk 1 above)
Comment (psychocoderHPC) [reply lib/isaac/isaac_version.hpp]: fixed

**Review by PrometheusPi — APPROVED** [review]

**Pull request merged**